app/Http/Controllers/Consumables/ConsumableCheckoutController.php

```diff
@@ -69,6 +70,12 @@ public function store(Request $request, $consumableId)
             'note' => $request->input('note'),
         ]);
 
+        $checkedout=DB::table('consumables_users')->where('consumable_id', '=', $consumable->id)->count();
+        $available=DB::table('consumables')->where('id', '=', $consumable->id)->first('qty');
```

Comment:
Same for this one :)

https://github.com/snipe/snipe-it/blob/c24b398b6f4c768e2531e119235f1013351075c5/app/Models/Consumable.php#L319-L326

Reply:
used! thanks for showing me this 🙂